cortico.js

```diff
@@ -1558,6 +1606,41 @@ async function setupPreferredPharmacies() {
   }
 }
 
+
+async function init_diagnostic_viewer_button() {
+  const notesField = document.querySelector("textarea[name='notes']");
+  var notesValue = notesField.textContent;
+  console.log("echo", notesValue)
+
+  var last_button = document.querySelector("#cortico").parentNode;
+  last_button.parentNode.innerHTML +=
+    "<button class='cortico-btn' id='diagnostic-viewer-btn' style='color:white; background-color:blue'>Diagnostic Viewer</button>";
+
+
+  const corticoDiagnosticViewBtn = document.getElementById("diagnostic-viewer-btn")
+  function update_diagnostic_button_visibility() {
+    notesValue = notesField.textContent;
+    console.log("notes value", notesValue.toLowerCase())
+
+    corticoDiagnosticViewBtn.style.visibility =
+    notesValue.includes("-- Cortico data below, do not change!") ? "visible" : "hidden";
+  }
+
+  async function open_diagnostic_viewer(e) {
+    e.preventDefault();
+
+    const appt_no = getQueryStringValue("appointment_no");
+    const diagnostic_response = await getDiagnosticFromCortico(appt_no, notesValue)
+    const diagnostic_text = await JSON.parse(diagnostic_response.text())
+    await showDiagnosticResults(diagnostic_text)
+  }
+
+  update_diagnostic_button_visibility();
+
+  corticoDiagnosticViewBtn.addEventListener("click", open_diagnostic_viewer)
```

Comment:
If there is no clinic name set, we should show a message "You must set your Coritco clinic to use this feature." with a link to go to that area.